packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -51,7 +51,7 @@
                         name="mobile-number"
                         maxlength="16"
                         input-type="tel"
-                        :label-text="$t('labels.mobileNumber')"
+                        :label-text="$t('formFields.customer.mobileNumber.label')"
```

Comment:
Could we use some variables so that we don't need to type out the full qualifications each time?

Reply:
This will be done in the next pr https://github.com/justeat/fozzie-components/compare/f-checkout-translations...f-checkout-guest-form-field?expand=1